lib/MusicBrainz/Server/Form/Role/ValidateUsername.pm

```diff
@@ -0,0 +1,22 @@
+package MusicBrainz::Server::Form::Role::ValidateUsername;
```

Comment:
Or maybe lib/MusicBrainz/Server/Form/Utils.pm instead, since the Validation module doesn't really contain anything that operates on FormHandler fields directly right now. Having a generic "Form::Role::Username" still makes sense too IMHO.